TaskList/app/assets/images/Magical Task List.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
I have absolutely no clue what is going on with the files in `TaskList/app/assets/images/` from your commit 69baee3215899544f7bb17c104ecc532c180c099. This is hundreds, possibly thousands of lines of JavaScript code. Could you please explain/justify adding it into your project?